docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       SPLUNK_COLLECTOR_URL: ${SPLUNK_COLLECTOR_URL}
       SPLUNK_TOKEN: ${SPLUNK_HEC_TOKEN}
     ports:
-      - "5000:80"
+      - "5000:8080"
```

Comment:
like that, + most platforms does not allow you to run  on 80

Reply:
Redhat container runs 8080 default.  Are you asking for a change here?

Reply:
Nope, just saying great!